airbyte-integrations/connectors/destination-s3-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/s3_v2/S3V2WriteTest.kt

```diff
@@ -150,19 +150,7 @@ class S3V2WriteTestAvroUncompressed :
         preserveUndeclaredFields = false,
         allTypesBehavior = StronglyTyped(),
     ) {
-    @Disabled("Not yet working")
-    @Test
-    override fun testContainerTypes() {
-        super.testContainerTypes()
-    }
-
-    @Disabled("Not yet working")
-    @Test
-    override fun testUnions() {
-        super.testUnions()
-    }
-
-    @Disabled("Not yet working")
+    @Disabled("Test does not support `stringifyShemalessObjects == true`")
```

Comment:
blegh, submitted https://github.com/airbytehq/airbyte-internal-issues/issues/10628 to fix this